hnn_core/network.py

```diff
@@ -1253,23 +1253,42 @@ def add_connection(self, src_gids, target_gids, loc, receptor,
 
         self.connectivity.append(deepcopy(conn))
 
-    def clear_connectivity(self):
-        """Remove all connections defined in Network.connectivity
-        """
-        connectivity = list()
+    def clear_connectivity(self, src_types=None):
+        """Remove connections with src_type in Network.connectivity."""
```

Comment:
can you document `src_types` ?

Reply:
also is `clear_connectivity` tested?

Reply:
-Sure will add the documentation for src_types.
- Yes it is tested in all three tests. First test checks the base conditions where no arguments are passed. Here it successfully deletes all cell connections apart from those originating from external drives
- The other two tests it tests for deletion of connections from specific source types.